src/ThreatMonitoring/SecurityScanClient.php

```diff
@@ -44,6 +44,37 @@ public function getById($id)
     }
 
     /**
+     * Get group of scans by any one scan id.
+     *
+     * @param $id
+     * @param int $page
+     * @param int $perPage
+     * @param array $filters
+     * @return SecurityScan
+     */
+    public function getGroupById($id, $page = 1, $perPage = 15, $filters = [])
+    {
+        $page = $this->paginatedRequest('v1/scans/group/' . $id, $page, $perPage, $filters);
+
+        $page->serializeWith(function ($item) {
+            return $this->serializeResponse($item);
+        });
+
+        return $page;
+    }
+
+    /**
+     * Get a security scans total
+     * @return SecurityScan
+     */
+    public function getAll()
+    {
+        $response = $this->get('v1/scans');
+        $body = $this->decodeJson($response->getBody()->getContents());
+        return $this->serializeResponse($body->data);
```

Comment:
Does the `scans` endpoint not return an array of scans? If so, we might need to array_map or loop through this and serialize each item individually